wordpress-mobile/WordPress-iOS
# Use an image to indicate new notifications

Fixes #7204 

To test:
Install this version.
Get some notifications, got to the Notification tab, go back.
Check that the right image is displayed at all times

Needs review: @koke
